### ISSUE TITLE
Can this be used for an enterprise identity provider (non-Social provider)?

### ISSUE THREAD

**Issue by dosstx** [dgrubelic/vue-authenticate#234]
Hi, 
 Is this package only for using social providers (Google, Github, etc.) ? Could I use my own company's identity provider ?

**Comment by dgrubelic** [comment]
No, you can add your own provider. I think there were some examples in issues here where people built apps with their own providers.

https://github.com/dgrubelic/vue-authenticate/blob/master/src/options.js#L165
https://github.com/dgrubelic/vue-authenticate/blob/master/src/options.js#L174

**Comment by dosstx** [comment]
@dgrubelic Thanks for the info! Will check it out...

**Comment by dosstx** [comment]
@dgrubelic It's possible I have something working....but not quite there yet. 

Can you explain what this property does?

https://github.com/dgrubelic/vue-authenticate/blob/f4aa8cb9a416842557213edaab19a30085b0a0aa/src/options.js#L176


I am unsure what to put there. Thanks.

**Comment by dgrubelic** [comment]
This is the endpoint on your server app which will initiate and be used for oauth2 authentication. You handle all loading users other stuff there. There was example app how to implement that on server, but I removed it and will make example app that uses this library.

**Comment by dosstx** [comment]
@dgrubelic OK, any examples you can provide would be awesome! I still don't quite understand what to put there. Is it one of these endpoints?
![image](https://user-images.githubusercontent.com/6342102/120858665-b894aa80-c548-11eb-874c-6fc77e76f175.png)

It's not the OAuth Authorization URL, right? I'm already providing that for the `authorizationEndpoint` property in the providers object.

**Comment by dosstx** [comment]
OK, everything seems to work locally at least...but still confused about difference between these two properties in the providers object:

`authorizationEndpoint` and `url`. 

Thanks for the library and I look forward to any future updates.

**Comment by ajmas** [comment]
@dosstx did you resolve your issue.

Have made a draft explanation the properties in this ticket: #241

**Comment by dosstx** [comment]
I haven't tried it yet but that draft explanation looks very helpful! Thanks.
